packages/ui/src/composables/useSelectableList.ts

```diff
@@ -36,7 +36,10 @@ export function useSelectableList (props: ExtractPropTypes<typeof useSelectableL
 
   const getText = (option: SelectableOption) => getOptionProperty(option, props.textBy)
   const getDisabled = (option: SelectableOption) => getOptionProperty(option, props.disabledBy)
-  const getTrackBy = (option: SelectableOption) => getOptionProperty(option, props.trackBy)
+  const getTrackBy = (option: SelectableOption) => {
+    if (props.trackBy) { return getOptionProperty(option, props.trackBy) }
+    return getValue(option)
+  }
```

Comment:
I think we should treat other methods in the same way.

But maybe it would be better to rework the function:
```ts
const getOptionProperty = (option: SelectableOption, selector: StringOrFunction) => {
  if (isStringOrNumber(option)) {
    return option
  }

  return selector
    ? getValueByKey(option, selector)
    : getValue()
}
```

Reply:
Discussed.